### ISSUE TITLE
snakepipes 2.0.1

### ISSUE THREAD

**Issue by AdrijaK** [maxplanck-ie/snakepipes#609]
It seems that ChIP-seq pipeline ignores the sample order provided in samples.tsv to determine which sample is "treatment" and which is "control". 

H3K27ac_original.tsv:
```
name	condition
H3K27ac-dox-rep1	dox
H3K27ac-dox-rep2	dox
H3K27ac-dox-rep3	dox
H3K27ac-veh-rep1	veh
H3K27ac-veh-rep2	veh
H3K27ac-veh-rep3	veh
```
H3K27ac_flipped.tsv:
```
name	condition
H3K27ac-veh-rep1	veh
H3K27ac-veh-rep2	veh
H3K27ac-veh-rep3	veh
H3K27ac-dox-rep1	dox
H3K27ac-dox-rep2	dox
H3K27ac-dox-rep3	dox
```

output in both cases gives the same number of up- and down- bound regions: 

```bash
wc -l CSAW_MACS2_H3K27ac_original/Filtered*.bed
```
```
CSAW_MACS2_H3K27ac_original/Filtered*.bed
  13190 CSAW_MACS2_H3K27ac_original/Filtered.results.bed
   3321 CSAW_MACS2_H3K27ac_original/Filtered.results.DOWN.bed
      2 CSAW_MACS2_H3K27ac_original/Filtered.results.MIXED.bed
   9867 CSAW_MACS2_H3K27ac_original/Filtered.results.UP.bed
```

```bash
wc -l CSAW_MACS2_H3K27ac_flipped/Filtered*.bed
```
```
  13190 CSAW_MACS2_H3K27ac_flipped/Filtered.results.bed
   3321 CSAW_MACS2_H3K27ac_flipped/Filtered.results.DOWN.bed
      2 CSAW_MACS2_H3K27ac_flipped/Filtered.results.MIXED.bed
   9867 CSAW_MACS2_H3K27ac_flipped/Filtered.results.UP.bed
```

[run_log_for_original.txt](https://github.com/maxplanck-ie/snakepipes/files/4444786/run_log_for_original.txt)
[run_log_for_flipped.txt](https://github.com/maxplanck-ie/snakepipes/files/4444785/run_log_for_flipped.txt)

**Comment by katsikora** [comment]
I can reproduce your issue. The design matrix is correct as visible in the CSAW html report, the issue must be further downstream. I will have a look.

**Comment by katsikora** [comment]
The fix for this issue will be available in the upcoming release.

**Comment by AdrijaK** [comment]
Thank you!

**Comment by katsikora** [comment]
Fixed in 2.1.0. Feel free to reopen if anything comes up.
